### PR TITLE
Support v12 in check tool

### PR DESCRIPTION
**Pull request by kmaehashi**

### CODE REVIEW

File: check_release_assets.py [hunk 1]
```diff
@@ -85,7 +102,7 @@ def get_expected_wheel_basename(project, version, abi, arch):
 
 
 def verify(project, expected, actual):
-    print('🔵 Project: {}'.format(project))
+    print('🔵 PyPI: {}'.format(project))
```
Comment (asi1024) [inline check_release_assets.py:105]: This function is also called from L150. https://github.com/cupy/cupy-release-tools/pull/295/files#diff-bca831d24e956928fc7f3bf8bc599b9097cc39f205b135028a28433fd36db964R150

Comment (kmaehashi) [reply check_release_assets.py:105]: Ah, nice catch!
```suggestion
    print('🔵 Project: {}'.format(project))
```

File: check_release_assets.py [hunk 2]
```diff
@@ -14,18 +14,26 @@
...
+_v12_rocm_matrix = list(itertools.product(
+    (CP37, CP38, CP39, CP310), (LINUX,)))
+
+_v11_cuda_matrix = list(itertools.product(
+    (CP37, CP38, CP39, CP310), (LINUX_V11, WINDOWS)))
 _v11_aarch64_matrix = list(itertools.product(
     (CP37, CP38, CP39, CP310), (LINUX_AARCH64,)))
 _v11_rocm_matrix = list(itertools.product(
-    (CP37, CP38, CP39, CP310), (LINUX,)))
+    (CP37, CP38, CP39, CP310), (LINUX_V11,)))
```
Comment (asi1024) [inline check_release_assets.py:36]: This PR changes also v11 matrices. Is it intended?

Comment (kmaehashi) [reply check_release_assets.py:36]: Yes. v12 (`LINUX`) will use `manylinux2014` (#282). v11 keep using `manylinux1`.

**Pull request merged**